e2e/setup/setup_test.go

```diff
@@ -128,6 +128,29 @@ func Test_CR(t *testing.T) {
 	)
 }
 
+// Test_PriorityClass will get the PriorityClass name from the StatefulSet and checks if a PriorityClass with that
+// name exists in the cluster.
+func Test_PriorityClass(t *testing.T) {
+	ctx := context.TODO()
+
+	err := Retry(attempts, interval, func() error {
+		sts, stsErr := clientSet.AppsV1().StatefulSets(NamespaceName).Get(ctx, STSName, metav1.GetOptions{})
+		if stsErr != nil {
+			return stsErr
+		}
+
+		pcName := sts.Spec.Template.Spec.PriorityClassName
+		if len(pcName) < 1 {
```

Comment:
this makes the test kinda pointless. the problem is, that the latest release does not have pc name set so the upgrade test will fail, for the current release.

Reply:
thats okay, just also check that the pcName == "nats-manager-priority-class"

Reply:
good point, added that